src/codeflare_sdk/utils/generate_yaml.py

```diff
@@ -314,6 +343,12 @@ def main():  # pragma: no cover
         default="default",
         help="Set the kubernetes namespace you want to deploy your cluster to. Default. If left blank, uses the 'default' namespace",
     )
+    parser.add_argument(
+        "--priority",
+        required=False,
+        default="low",
+        help="Set the priority of the cluster. Default is 'low'. Options are 'low', 'default', 'high'",
```

Comment:
We should mention the word `dispatch priority`

Reply:
and to be consistent in the code change variable name priority to dispatch priority and also the args name